src/bundle/Resources/views/fieldtype/fielddefinition_settings.html.html.twig

```diff
@@ -1,17 +1,17 @@
 {% trans_default_domain "fielddefinition" %}
 
 {% block ezcontentquery_settings %}
-    <li class="ez-fielddefinition-setting query-type">
-        <div class="ez-fielddefinition-setting-name">{{ 'ezcontentquery.query_type.label'|trans|desc("Query type:")}}</div>
-        <div class="ez-fielddefinition-setting-value">
+    <li class="ibexa-fielddefinition-setting query-type">
```

Comment:
```suggestion
    <li class="ibexa-field-definition-setting query-type">
```
?

Reply:
Outside scope, I don't want to do any more changes than renaming ez/ezplatform to ibexa it his task